### PR TITLE
Fix wrong env variable name

### PR DESCRIPTION
**Pull request by dellamowica**
fix a copy/paste typo

### CODE REVIEW

**Comment by javiereguiluz** [comment]
I see this might look a bit confusing, but in this very same example, we include the following a few lines above:

```php
if (isset($_ENV['BOOTSTRAP_CLEAR_CACHE_ENV'])) {
```

So, I think this might be correct. But it's true that it can be confusing because "Bootstrap" looks like something related to the Bootstrap CSS framework instead of the Symfony bootstrap process 🤔

**Comment by dellamowica** [comment]
yes it's a bit confusing, like why clearing the cache of another environnement than the testing environnement :D 
I understood that BOOTSTRAP_CLEAR_CACHE_ENV was more like a boolean toggling a clear cache during tests if set in the phpunit.xml file

**Comment by javiereguiluz** [comment]
I'm not sure what to do here. Let's ask @wouterj to see if he can come up with a better name or has any other suggestion. Thanks!

**Comment by wouterj** [comment]
I'm fine with not using the custom variable here. Using a different environment for the application is not that common, so let's make the example easier (if you need advanced stuff, you can probably come up with the solution yourself).

But this means we must also remove the if statement lines, and lines 51 - 67 (where we talk about configuring the custom env var).

Can you update this PR with these changes, @dellamowica? (you can do so by creating a new commit on your [`patch-1` branch](https://github.com/dellamowica/symfony-docs/tree/patch-1))

**Comment by dellamowica** [comment]
done ;-)

File: testing/bootstrap.rst [hunk 1]
```diff
@@ -48,22 +46,3 @@ You can modify this file to add custom logic:
...
-        <php>
-            <env name="BOOTSTRAP_CLEAR_CACHE_ENV" value="test"/>
-            <!-- ... -->
-        </php>
-
-        <!-- ... -->
-    </phpunit>
-
-Now, when running ``vendor/bin/phpunit``, the cache will be cleared
-automatically by the bootstrap file before running all tests.
```
Comment (wouterj) [inline testing/bootstrap.rst:69]: I think we would like to keep this paragraph.

**Review by wouterj — APPROVED** [review]
Thank you!

I've left one last suggestion, but we can also fix it during the merge.

File: testing/bootstrap.rst (hunk 1 above)
Comment (javiereguiluz) [reply testing/bootstrap.rst:69]: Done while merging. Thanks!

**Comment by javiereguiluz** [comment]
@dellamowica thanks for this contribution and for your patience during the review process (thanks to reviewers too!). Also, congrats on your first Symfony Docs contribution 🎉

**Pull request merged**